tests/test_callbacks.py

```diff
@@ -0,0 +1,33 @@
+import os
+from pathlib import Path
+from dash_uploader import UploadStatus
+from app.app import disable_tabs
+from app.app import upload_data
+from . import DATA_DIR
+
+
+MOCK_FILE_PATH = Path(DATA_DIR, "mock_obj_data.pkl")
+
+
+def test_upload_data():
+    # Create an UploadStatus object
+    status = UploadStatus(
+        uploaded_files=[MOCK_FILE_PATH], n_total=1, uploaded_size_mb=5.39, total_size_mb=5.39
+    )
+    upload_string, path_string = upload_data(status)
+
+    # Check the result
+    assert upload_string == f"Successfully uploaded: {os.path.basename(MOCK_FILE_PATH)} [5.39 MB]"
```

Comment:
```suggestion
    assert upload_string == f"Successfully uploaded: {MOCK_FILE_PATH.name} [5.39 MB]"
```

See the table https://docs.python.org/3.10/library/pathlib.html#correspondence-to-tools-in-the-os-module